CHANGELOG.next.asciidoc

```diff
@@ -59,9 +58,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...main[Check the HEAD dif
 - Fix compatibility with ECS by renaming `source` log key to `source_file` {issue}30667[30667]
 - Fix add_kubernetes_metadata matcher: support rotated logs when `resource_type: pod` {pull}30720[30720]
 - Update documentation for accessing `last_response.url.params` in httpjson input. {pull}30739[30739]
-
-*Filebeat*
-
+- Mitigates data duplication on restart when filestream inputs did not have an ID set. Setting an ID for those
+  inputs will fix the problem without generating any data duplication{issue}30061[30061]
```

Comment:
Close, I don't think we have mitigated it, but we have allowed mitigating it with a config change. What about something like:

```suggestion
- Allow fixing data duplication on restart when filestream inputs did not have an ID set. Setting an ID for filestream
  inputs will fix the problem without generating any data duplication{issue}30061[30061]
```